src/analysis/retail/evoker/shared/constants.ts

```diff
@@ -72,6 +74,7 @@ export const FAN_THE_FLAMES_INCREASE = 1;
 export const RED_HOT_INCREASE = 0.2;
 
 export const PRIMACY_HASTE_PER_STACK = 3;
+export const THREAD_OF_FATE_BASE_DURATION = 10;
```

Comment:
This should also be in milliseconds.
```suggestion
export const THREAD_OF_FATE_BASE_DURATION_MS = 10_000;
```